lessons/micropython/mini-workshop/index.md

```diff
@@ -0,0 +1,319 @@
+
+!!! note ""
+
+    Tento workshop nejde projít jen tak z domu – je potřeba speciální příprava.
+    Pokyny pro organizátory najdeš
+    na [podstránce]({{ subpage_url('organizers') }}).
+
+Jedna varianta Pythonu, MicroPython, běží na malých zařízeních.
+Taková zařízení si teď můžeš vyzkoušet.
+
+Jak na to?
+
+
+## Otevření příkazové řádky
+
+<img src="{{ static('icon_terminal.png')}}" style="float:right;height:5em;">
+
+Klikni na *Activities*; potom vyber z levého panelu *Terminal* (ikonka
+černého okýnka).
```

Comment:
To bych dělal až k tomu bude Ansible playbook, kde to bude možné potunit všechno najednou. Zatím mám přístup k NB s anglickýma Fedorama.